### PR TITLE
FEAT: OmniSciDB Added fragment_size to table creation

### PR DESCRIPTION
**Pull request by semelianova**

### CODE REVIEW

File: ibis/omniscidb/client.py [hunk 1]
```diff
@@ -931,13 +937,24 @@ def create_table(
         obj : TableExpr or pandas.DataFrame, optional
           If passed, creates table from select statement results
         schema : ibis.Schema, optional
+        table_name : str
```
Comment (jreback) [inline ibis/omniscidb/client.py]: looks like obj got repeated here

File: ibis/omniscidb/ddl.py [hunk 2]
```diff
@@ -120,21 +120,29 @@ def compile(self):
...
+    ):
         self.table_name = table_name
         self.database = database
         self.schema = schema
         self.max_rows = max_rows
+        self.fragment_size = fragment_size
 
     @property
     def with_params(self):
         """Return the parameters for `with` clause.
```
Comment (jreback) [inline ibis/omniscidb/ddl.py]: can you type return values

File: ibis/omniscidb/ddl.py [hunk 3]
```diff
@@ -120,21 +120,29 @@ def compile(self):
 class CreateTableWithSchema(CreateTable):
     """Create Table With Schema class."""
 
-    def __init__(self, table_name, schema, database=None, max_rows=None):
+    def __init__(
+        self,
+        table_name,
```
Comment (jreback) [inline ibis/omniscidb/ddl.py]: type if possible

File: ibis/omniscidb/ddl.py (hunk 2 above)
Comment (semelianova) [reply ibis/omniscidb/ddl.py]: Return value type is below, 145 line

Comment (anmyachev) [reply ibis/omniscidb/ddl.py]: @semelianova you should do something like that:
```with_params(self) -> [str or something else]```

Comment (jreback) [reply ibis/omniscidb/ddl.py]: return value is Dict[str, Any]

**Comment by jreback** [comment]
can you rebase

**Comment by semelianova** [comment]
> typing comment, ping on green.

What do you mean? Comment to pr or something else?

**Comment by jreback** [comment]
> > typing comment, ping on green.
> 
> What do you mean? Comment to pr or something else?

meaning when the CI is green (as now), pls make a comment

**Comment by jreback** [comment]
thanks @semelianova

File: docs/source/release.rst [hunk 4]
```diff
@@ -10,6 +10,7 @@ Release Notes
 * :feature:`2126` Add translation rules for isnull() and notnull() for pyspark backend
 * :feature:`2062` Implement read_csv for omniscidb backend
 * :feature:`2060` Add initial support for ibis.random function
+* :support:`2107` Added fragment_size to table creation for OmniSciDB
```
Comment (xmnlab) [inline docs/source/release.rst]: I am wondering if it should be `feature` instead of `support` ...

**Pull request merged**